specifyweb/specify/migrations/0002_geo.py

```diff
@@ -67,7 +68,10 @@ def create_default_collection_types():
         # Update CollectionObjects' collectionobjecttype for the discipline
         Collectionobject.objects.filter(collection=collection).update(collectionobjecttype=cot)
         collection.collectionobjecttype = cot
-        collection.save()
+        try:
+            collection.save()
+        except BusinessRuleException as e:
+            continue
```

Comment:
Ya, I ran into this error for one of the databases.  I'll try to recreate it again, I believe it was in the hujinnhc test panel database migration.  A complete solution that should be done is to handle and resolve the business rule exception so that the collection field is not null.